investigator/overlays/test/kustomization.yaml

```
@@ -7,6 +7,7 @@ resources:
   - route.yaml
   - service.yaml
   - thoth-notification.yaml
+  - deploymentconfig.yaml
```

Add this under the key patchesStrategicMerge, as this has to patch into the base dc.

```suggestion
patchesStrategicMerge:
  - deploymentconfig.yaml
```

---

Got it, ty